src/components/popovers/menu.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta, Preview, Story, Props } from "@storybook/addon-docs/blocks";
+import {
+  withKnobs,
+  array,
+  text,
+  boolean,
+  optionsKnob as options
+} from "@storybook/addon-knobs";
+import { action } from '@storybook/addon-actions';
+
+import Menu from './Menu';
+import MenuItem from './MenuItem';
+
+import Button from '../interactive/Button';
+
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+
+import CloneIcon from 'lib/icons/react/CloneIcon';
+import TrashIcon from 'lib/icons/react/TrashIcon';
+import EditIcon from 'lib/icons/react/EditIcon';
+import FundingIcon from 'lib/icons/react/FundingIcon';
+
+<Meta title="Popovers|Menu" component={Menu} decorators={[withKnobs]} />
+
+# Menu
+
+Renders a basic keyboard navigable dropdown menu following the [WAI-ARIA pattern for "menubutton"](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton).
+
+### When to use Menu
+Only use a `Menu` for dropdowns that contain actions the user can take. If a dropdown includes any other type of content (forms, arbitrary content), use a `Popover` component instead.
+
+### Positioning
+The menu will automatically position above or below the trigger element based on its position in the viewport.
+
+<Story name="Knobs" parameters={{ docs: { disable: true }}}>
+  <div style={{ padding: '2rem' }}>
```

Comment:
This is a nitpick, but do we want to be using rem here?

Reply:
no reason, really - just wanted to arbitrarily position the knobs story away from the edge of the canvas so it's easier to see the dynamic positioning.